### PR TITLE
[Symfony] Try to get env parameter from container before throwing an exception

### PR DESCRIPTION
**Pull request by Steveb-p**
In regard to an issue raised on gitter.

image

### CODE REVIEW

**Comment by makasim** [comment]
The issue was properly fixed in 0.9 version. Unfortunately, it is not possible to fix it in 0.8 version.  
There is another attempt to do it https://github.com/php-enqueue/enqueue-dev/pull/533

File: pkg/enqueue/Symfony/DefaultTransportFactory.php [hunk 1]
```diff
@@ -161,7 +162,11 @@ private function resolveDSN(ContainerBuilder $container, $dsn)
             $matches = [];
             if (preg_match('/%env\((.*?)\)/', $dsn, $matches)) {
                 if (false === $realDsn = getenv($matches[1])) {
-                    throw new \LogicException(sprintf('The env "%s" var is not defined', $matches[1]));
+                    try {
+                        $realDsn = $container->getParameter($matches[1]);
```
Comment (makasim) [inline pkg/enqueue/Symfony/DefaultTransportFactory.php]: shouldn't it be `$dsn` instead of `$matches[1]`

Comment (makasim) [reply pkg/enqueue/Symfony/DefaultTransportFactory.php]: @Steveb-p ping ^^

Comment (Steveb-p) [reply pkg/enqueue/Symfony/DefaultTransportFactory.php]: I'll take a look at it tonight :)

**Comment by makasim** [comment]
@Steveb-p is there any news?

**Comment by makasim** [comment]
The issue is properly fixed in 0.9 version which is soon to be released.

**Pull request closed without merging**